string box_name
bool add
bool attach
geometry_msgs/PoseStamped box_pose
float32[] box_size
bool exit
---
bool success
